pkgs/by-name/ro/rollo-x1038/package.nix

```
@@ -0,0 +1,30 @@
+{
+  stdenv,
+  cups,
+  lib,
+}:
+
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
Some attrs will be reused